Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -331,7 +331,7 @@ class BrowserViewController: UIViewController {
 
     private func showHomePanelController(#inline: Bool) {
         homePanelIsInline = inline
-
+        
```

Comment:
Please undo this change. You'll want to enable both "Automatically trim trailing whitespace" and "Including whitespace-only lines" preferences in Xcode (under Text Editing)